crates/experimentation_platform/src/api/experiments/handlers.rs

```diff
@@ -794,29 +841,47 @@ async fn update_overrides(
 
     let http_client = reqwest::Client::new();
     let url = state.cac_host.clone() + "/context/bulk-operations";
-    let headers_map = construct_header_map(tenant.as_str(), custom_headers.config_tags)?;
+    let user_str = serde_json::to_string(&user).map_err(|err| {
+        log::error!("Something went wrong, failed to stringify user data {err}");
+        unexpected_error!(
+            "Something went wrong, failed to stringify user data {}",
+            err
+        )
+    })?;
+    let extra_headers = vec![
+        ("x-user", Some(user_str)),
+        ("x-config-tags", custom_headers.config_tags),
+    ]
+    .into_iter()
+    .filter_map(|(key, val)| val.map(|v| (key, v)))
+    .collect::<Vec<_>>();
+
+    let headers_map = construct_header_map(tenant.as_str(), extra_headers)?;
 
     let response = http_client
         .put(&url)
         .headers(headers_map.into())
         .header(
-            "Authorization",
-            format!("{} {}", user.get_auth_type(), user.get_auth_token()),
+            header::AUTHORIZATION,
+            format!("Internal {}", state.superposition_token),
```

Comment:
What is internal auth?

Reply:
This is not a supported authentication scheme
https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes

Reply:
https://www.iana.org/assignments/http-authschemes/http-authschemes.xhtml